src/main/java/io/vlingo/actors/Stage.java

```diff
@@ -122,23 +122,6 @@ public Stage(final World world, final AddressFactory addressFactory, final Strin
     return actor.protocolActor();
   }
 
-  public <T> T actorThunkFor(
-      final Class<T> protocol,
-      final Definition definition,
-      final Address address) {
-    final Mailbox actorMailbox = this.allocateMailbox(definition, address, null);
-    final ActorProtocolActor<T> actor =
-        actorProtocolFor(
-            protocol,
-            definition,
-            definition.parentOr(world.defaultParent()),
-            address,
-            actorMailbox,
-            definition.supervisor(),
-            definition.loggerOr(world.defaultLogger()));
-    return actor.protocolActor();
-  }
```

Comment:
Missed that I am overriding this in `Grid` to suspend the mailbox for resume (needs to happen before `Start` is delivered). I will look into alternative ways of doing that, but I might have to put it back in.